index.html

```diff
@@ -0,0 +1,233 @@
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <ul>
+                                <li class="project-language"> Html<li>
+                                <li class="project-language"> Bootstrap<li>
+                                <li class="project-language"> Ruby<li>
+                            </ul>
+                        </div>
+                        <a href="#" class="project-btn">See project</a>
+                        
+                     </div> 
+                     <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <ul>
+                                <li class="project-language"> Html<li>
+                                <li class="project-language"> Bootstrap<li>
+                                <li class="project-language"> Ruby<li>
+                            </ul>
+                        </div>
+                        <a href="#" class="project-btn">See project</a>
+                        
+                     </div> 
+                     <div class="project">
+                        <h2>Profesional Art Printing Data</h2>
+                        <p>A daily selection of privately personalized reads
+                             no accounts or sign-ups required. has been the industry's standard
+                        </p>
+                        <div class="languages">
+                            
+                            <ul>
+                                <li class="project-language"> Html<li>
+                                <li class="project-language"> Bootstrap<li>
+                                <li class="project-language"> Ruby<li>
+                            </ul>
+                        </div>
+                        <a href="#" class="project-btn">See project</a>
+                        
+                     </div>
+
+                  
+                 </div>
+                </section>
+
+            <section class="about">
+                <div class="description">
+                    <h2>About me</h2>
+                    <p>Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! 
+                    If you like what you see and have a project you need coded, don’t hestiate to contact me.
+                    </p>
+                    <a href="#">Get My Resume</a>
```

Comment:
- [ ] As mentioned earlier, please add `:hover` and `:active` stylings to your links.

Reply:
Alright, @TedLivist This has been noted.Thanks!